test-monitor/nitestmonitor.yml

```diff
@@ -1187,16 +1187,26 @@ paths:
             $ref: '#/definitions/AdvancedQuery'
         - in: query
           name: skip
+          description: >-
+            The number of results to skip when paging.
```

Comment:
Potential descriptions to replace this one with:
- The number of results to skip when returning results.
- The number of results to omit from the returned results.
- The number of results to not include in the returned results.

Reply:
What are your thoughts on `The number of matching results to skip when querying.`?

Reply:
I like that! 👍